ckanext/datarequests/controllers/ui_controller.py

```diff
@@ -37,8 +37,8 @@ def comment(self, id):
     def delete_comment(self, datarequest_id, comment_id):
```

Comment:
so not changing delete_comment to id also?

Reply:
No. This one uses the names consistently, plus it has two separate IDs that it needs to distinguish.